src/scripts/public/sites/cleanbc/patterns/landing-banners.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Takeover navigation manipulation.
+ */
+const domReady = () => {
+	/*
+	 * SafarIE bug requires 0ms timeout.
+	 */
+	setTimeout(function() {
+		const isHomeBanner = document.querySelectorAll('.home-cover-banner');
+		const isLandingBanner = document.querySelectorAll(
+			'.landing-cover-banner'
+		);
+
+		if (isHomeBanner.length || isLandingBanner.length) {
```

Comment:
Maybe a quick description of what your trying to accomplish here, for your future self, and the rest of us.

Reply:
Comment added.